src/søknad/steg/4-barnasbosted/BarnetsBostedLagtTil.tsx

```diff
@@ -54,9 +54,9 @@ const BarnetsBostedLagtTil: React.FC<Props> = ({
               {intl.formatMessage({ id: 'barnasbosted.element.andreforelder' })}
             </Label>
             <BodyShort>
-              {forelder.navn?.verdi
-                ? forelder.navn.verdi
-                : hentTekst('barnasbosted.kanikkeoppgiforelder', intl)}
+              {forelder.navn?.verdi === 'Ikke oppgitt'
```

Comment:
Både her og i `OmAnnenForelder.tsx` tror jeg engelsk går galt, ettersom det er `Not Registererd` på engelsk.

Jeg tror det er skummelt å sjekke verdi-innhold på denne måten når vi har forskjellige tekst-verdier

Reply:
Har gjort endringer da det ikke ble vist noe tekst her på førstegangssøknad og andre søknader enn BT.

Årsaken til at det går an å sjekke på 'Ikke oppgitt' er at dette alltid kommer fra backend på gjenbruk, da det ikke blir sendt med locale inn til den. Den klarer derfor ikke å sende den engelske teksten.